packages/contracts-periphery/contracts/foundry-tests/Optimist.t.sol

```diff
@@ -0,0 +1,371 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+/* Testing utilities */
+import { Test } from "forge-std/Test.sol";
+import { AttestationStation } from "../universal/op-nft/AttestationStation.sol";
+import { Optimist } from "../universal/op-nft/Optimist.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+contract Optimist_Initializer is Test {
+    address alice_admin = address(128);
```

Comment:
These values all up here can be constants, which will inline them in the bytecode rather than placing them in storage, except for the station + optimism bc they are set dynamically